lib/atscppapi/examples/Makefile.am

```diff
@@ -15,6 +15,8 @@
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
 
+SUBDIRS = experimental
```

Comment:
Ok. Originally I made it just an example and Brian Geffon said it should go in an experimental subdirectory. I'm happy to move it back if that's now the preferred option.